kogito-editors-js/packages/import-java-classes-component/src/components/ImportJavaClasses/Model/DMNSimpleType.ts

```diff
@@ -28,52 +28,52 @@ export enum DMNSimpleType {
   UNDEFINED = "<Undefined>",
 }
 /** It refers to "JavaBackedType" class determineTypeFromClass() method */
-export const JAVA_TO_DMN_MAP = {
+export const JAVA_TO_DMN_MAP = new Map([
```

Comment:
For the future, I'd suggest that this kind of association + enum of DMN Simple types could come in input to the component from the GWT editor.
Because ideally, the list of types could change (unlikely but still possible), and generally talking, it's better to have the definitions only in one place (GWT editor) instead of duplicating them (component + GWT editor).
Indeed, for example, with KOGITO-4588 even if I start with an enum like you, I will follow the approach that I described above.

Reply:
@vpellegrino Indeed, totally agree with you. Another reason for that change is: this component could be reused in BPMN editor too, so we need to parameterize that translation map in order to apply one for DMN and another one for BPMN.  I'll open a ticket to track it